plugins/reference-assets/android/src/main/res/layout/info.xml

```diff
@@ -33,4 +33,15 @@
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@+id/info_primary_info" />
 
+    <LinearLayout
```

Comment:
is there a reason for the choice of `LinearLayout` here? It looks like we only handle `footer` as a singular asset